### PR TITLE
MRG: Fix missing xlabel for epochs or raw.plot_psd when passing a subset o…

### LINKED ISSUE #5816: epochs.plot_psd loses axis labels
I would expect this code to work

```py
from mne.viz.tests.test_epochs import _get_epochs
import matplotlib.pyplot as plt
import mne

epochs = _get_epochs()

# subplot axis properly labeled
_, (ax1, ax2) = plt.subplots(nrows=2)
epochs.plot_psd(picks=mne.pick_types(epochs.info, meg='meg'), ax=[ax1, ax2])

# proper axis labels
epochs.plot_psd(picks=mne.pick_types(epochs.info, meg='mag'))

# missing labels
_, (ax1, ax2) = plt.subplots(nrows=2)
epochs.plot_psd(picks=mne.pick_types(epochs.info, meg='mag'), ax=ax1)
```
 
image

### PR DESCRIPTION
**Pull request by agramfort**
…f axis from a figure

closes #5816

### CODE REVIEW

File: mne/viz/utils.py [hunk 1]
```diff
@@ -3188,18 +3191,17 @@ def _plot_psd(inst, fig, freqs, psd_list, picks_list, titles_list,
...
             ax.set(xlim=[freqs[1] if freqs[0] == 0 else freqs[0], freqs[-1]])
             ax.get_xaxis().set_major_formatter(ScalarFormatter())
         else:
             ax.set(xlim=(freqs[0], freqs[-1]))
-        if make_label:
-            if ii == len(picks_list) - 1:
-                ax.set_xlabel('Frequency (Hz)')
-            ax.set(ylabel=ylabels[ii], title=titles_list[ii])
+        if xlabel:
+            ax.set_xlabel('Frequency (Hz)')
```
Comment (larsoner) [inline mne/viz/utils.py]: You errantly killed the `ylabels` setting

**Comment by larsoner** [comment]
... after `latest.inc` update :)

**Comment by larsoner** [comment]
pushed `latest.inc` update

File: mne/viz/utils.py (hunk 1 above)
Comment (agramfort) [reply mne/viz/utils.py]: fixed

Comment (larsoner) [reply mne/viz/utils.py]: I think this still needs to be nested in the `if make_label` code

IIRC I had it this way in the commit I pushed to your branch. You also removed a `_check_option('xscale', xscale, ('log', 'linear'))` that I had put in, was that intentional? If not, please restore it as it takes care of the problem where `xscale` could be anything.

**Comment by agramfort** [comment]
I lost your commits with a silly push -f. If you still have the
commits please push them here.

**Comment by agramfort** [comment]
thx @larsoner for your help

**Pull request merged**